extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -202,6 +203,7 @@
 		private static final String CONTENT_DISPOSITION = "contentDisposition";
 		private static final String DROP_URI_PATH = "dropURIPath";
 		private static final String CACHE_CONTROL = "cacheControl";
+		private static final String STRING_RAW_CONTENT = "stringRawContent";
```

Comment:
Okay, so it works like this (latest RFC 7159: https://tools.ietf.org/html/rfc7159):

> 9.  Parsers
>   A JSON parser transforms a JSON text into another representation.  A
>   JSON parser MUST accept all texts that conform to the JSON grammar.
>   A JSON parser MAY accept non-JSON forms or extensions.

> 2.  JSON Grammar
>   A JSON text is a sequence of tokens.  The set of tokens includes six
>   structural characters, strings, numbers, and three literal names.
>   A JSON text is a serialized value.  ...

> 3.  Values
>   A JSON value MUST be an object, array, number, or string, or one of
>   the following three literal names:
>   false null true

So Fabrizio you are right, there cannot be transferred **more than one array, object, value** per HTTP request. But a parser may be open for extensions, unlike the generator (_"The resulting text MUST strictly conform to the JSON grammar"_).